infrastructure/src/main/java/org/corfudb/infrastructure/DataStore.java

```diff
@@ -67,6 +68,7 @@ public DataStore(Map<String, Object> opts) {
     private LoadingCache<String, String> buildMemoryDS() {
         LoadingCache<String, String> cache = Caffeine
                 .newBuilder()
+                .recordStats()
```

Comment:
I'm not aware of Caffeine providing a way to disable/enable stats other than this builder function.  If we don't enable it here, there's no choice later.